modules/examiner/php/NDB_Menu_Filter_Form_examiner.class.inc

```diff
@@ -0,0 +1,308 @@
+<?php
+/**
+ * Examiner module menu filter form
+ *
+ * PHP Version 5
+ *
+ * @category Clinical
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@mail.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris
+ */
+
+require_once 'NDB_Menu_Filter.class.inc';
+require_once 'NDB_Form.class.inc';
+
+/**
+ * Examiner module menu filter form
+ *
+ * PHP Version 5
+ *
+ * @category Clinical
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@mail.mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris
+ */
+
+class NDB_Menu_Filter_Form_Examiner extends NDB_Menu_Filter_Form
+{
+    /**
+     * Checks if the user has access to examiner menu filter
+     *
+     * @return boolean
+     */
+    function _hasAccess()
+    {
+        $user = User::singleton();
+        return $user->hasPermission('examiner_view');
+    }
+
+    /**
+     * Sets the menu filter class variables.
+     *
+     * @return boolean
+     */
+    function _setupVariables()
+    {
+        $config           = NDB_Config::singleton();
+        $certification    = $config->getSetting('Certification');
+        $useCertification = $certification['EnableCertification'];
+
+        // base queries
+        $query     = " FROM examiners e LEFT JOIN psc ON (e.centerID=psc.CenterID)";
+        $certQuery = " LEFT JOIN certification c ON (c.examinerID = e.examinerID) 
+            LEFT JOIN test_names tn ON (c.testID = tn.ID)";
+
+        if ($useCertification == '1') {
+            $query .= $certQuery;
+        }
+
+        $query .= ' WHERE 1=1';
+
+        $user = User::singleton();
+        if (!$user->hasPermission('examiner_multisite')) {
+            $query .= " AND e.CenterID=" . $user->getData('CenterID');
+        }
+
+        // set the class variables
+        $this->columns      = array(
+                               'e.full_name as Examiner',
+                               'psc.Name as Site',
+                               'e.radiologist as Radiologist',
+                              );
+        $this->query        = $query;
+        $this->order_by     = 'e.full_name';
+        $this->headers      = array(
+                               'Examiner',
+                               'Site',
+                               'Radiologist',
+                              );
+        $this->validFilters = array(
+                               'e.full_name',
+                               'e.centerID',
+                               'COALESCE(e.radiologist, "0")',
```

Comment:
is the 0 in this case meant to be a string or an integer?

If it's an integer, there shouldn't be the quotes in SQL, and if it's a string, the SQL standard uses single quotes for strings, not double quotes (MySQL allows double quotes, but it's better to follow the standard more closely for broader compatibility..)